packages/syntax-core/src/RichSelect/RichSelectList.tsx

```diff
@@ -61,7 +62,7 @@ export type RichSelectListProps = RichSelectBoxProps & {
    */
   id?: string;
   /** Text shown above select box */
-  label?: string;
+  label: string;
```

Comment:
@jliotta `label` should still be required but it can be an empty string. When it is an empty string, the label won't render